projects/power_distribution/inc/pin_defs.h

```diff
@@ -142,3 +148,11 @@
   { .i2c_address = POWER_DISTRIBUTION_I2C_ADDRESS_0, .pin = PCA9539R_PIN_IO0_3 }
 #define REAR_PIN_SPARE_8_9_DSEL \
   { .i2c_address = POWER_DISTRIBUTION_I2C_ADDRESS_1, .pin = PCA9539R_PIN_IO1_2 }
+
+// fan ctrl pins - must be updated by HW
+#define REAR_PIN_ENC_VENT_THERM \
+  { GPIO_PORT_A, 0 }
+#define REAR_PIN_DCDC_THERM \
+  { GPIO_PORT_A, 1 }
+#define REAR_PIN_SMBALERT \
+  { GPIO_PORT_A, 2 }
```

Comment:
Currently hardware hasn't wired the SMB alert pin (as I'm sure you know), but PA2 is used by the current measurement module. Could you change this to something that's not used like PA0 again so that we don't accidentally mess up something else by reinitializing a GPIO pin?